README.md

```diff
@@ -202,6 +202,13 @@ $ build/uv_run_tests                        # shared library build
 $ build/uv_run_tests_a                      # static library build
 ```
 
+Or alternatively.
+
+```bash
+$ cmake -S . -B build
```

Comment:
I'm reasonably sure I didn't initially document it like that because cmake 3.0 doesn't support those flags but 3.10 is the minimum now and if they work with that version, you can just update the previous section.

Reply:
Done